case-lib/lib.sh

```diff
@@ -47,11 +47,15 @@ func_lib_setup_kernel_last_line()
 SOF_LOG_COLLECT=0
 func_lib_start_log_collect()
 {
-    local is_etrace=${1:-0} ldcFile
-    ldcFile=/etc/sof/sof-$(sof-dump-status.py -p).ldc || {
-        >&2 dlogw "sof-dump-status.py -p to query platform failed"
-        return
-    }
+    local is_etrace=${1:-0} ldcFile="$SOFLDC"
+    if [[ -z "$SOFLDC" ]]; then
```

Comment:
It's easier to read if you just use `$ldcFile` everywhere, I mean:
```shell
if [[ -z "$ldcFile" ]]; then
```

Reply:
I meant "everywhere in this function", sorry I wasn't clear. I meant this:
```shell
    local is_etrace=${1:-0} ldcFile="$SOFLDC"
    if [[ -z "$ldcFile ]]; then
```
Anyway I'm fine with `$ldcFile` in the interface too.

Reply:
thanks, your review helps a lot. fixed